graphene-pore/graphene-pore.ipynb

```diff
@@ -0,0 +1,167 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mbuild as mb\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from foyer import Forcefield\n",
+    "from calc_numberdensity import calc_number_density"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "water = mb.load('files/tip3p.mol2')\n",
+    "water.name = 'SOL'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c_ff = Forcefield('files/carbon.xml')\n",
+    "spce = Forcefield('files/spce.xml')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system = mb.recipes.GraphenePoreSolvent(pore_depth=2,side_dim=2,\n",
+    "                    n_sheets=3,pore_width=1.2,x_bulk=1.5,\n",
+    "                    solvent=water,n_solvent=600)\n",
+    "box = mb.Box(system.periodicity)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "water = mb.Compound()\n",
+    "graphene = mb.Compound()\n",
+    "for child in system.children:\n",
+    "    if child.name == 'SOL':\n",
+    "        water.add(mb.clone(child))\n",
+    "    else:\n",
+    "        graphene.add(mb.clone(child))\n",
+    "\n",
+    "water_pmd = spce.apply(water, residues='SOL')           \n",
+    "pore_pmd = c_ff.apply(graphene)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system = water_pmd + pore_pmd\n",
+    "system.box[:3] = box.maxs * 10.0\n",
+    "system.save('init.gro', overwrite=True, combine='all')\n",
+    "system.save('init.top', overwrite=True, combine='all')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gmx grompp -f files/nvt.mdp -c init.gro -p init.top -o nvt.tpr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gmx mdrun -deffnm nvt"
```

Comment:
personal preference but I like `-v` to show intermediate progress

Reply:
Yeah I'm unsure of this.  I removed `-v` yesterday because the log was making the notebook really long, but I could see the case having it.